custom/icds_reports/ucr/reports/testing/mpr_2bi_preg_delivery_death_list.json

```diff
@@ -88,6 +88,18 @@
       }
     ],
     "columns": [
+      {
+        "display": "dead_preg_count (for filter only)",
+        "column_id": "dead_preg_count",
+        "type": "sum_when",
+        "aggregation": "sum",
+        "calculate_total": true,
+        "whens": {
+          "female_death_type IS NOT NULL AND age_at_death_yrs >= 11": 1
+        },
+        "else_": 0,
+        "visible": false
+      },
```

Comment:
this is the only thing i'm uncertain of. attempting to mimic [this column](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/ucr/data_sources/person_cases_v2.json#L1154-L1195). I wasn't sure whether `IS NOT NULL` was right, or if I should also check for empty strings, or if the logic is wrong and it should be `= 'pregnant'` which would make more sense given the name of the column...

Reply:
I'm not sure. I've never gone through each report column and verified that the naming makes sense. (felt like that would cause a lot of trouble since the mobile app already depended on them when I came on) 

What about the parts of the column for sex = 'F' and closed = true?

Reply:
Good question. I was looking at some of the others, e.g. [dead_preg_resident_count](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/ucr/data_sources/person_cases_v2.json#L1090-L1121) which doesn't have the female expression, and also comparing to [indicators with the same name in reports](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/ucr/reports/mpr_2a_person_cases.json#L248-L258) and it kind of seemed optional. 

My guess: `female_death_type` existing implies closed and female based on logic maintained in the app, and so the other criteria are also implied. And the original report author was inconsistent in whether those conditions were made explicit or not.

These are a bit of a mess, hey?

Reply:
Cool. that makes sense.

Agreed that these are inconsistent, and it can make it hard to reason about them. I've usually defaulted to keeping the exact same behavior, but even though we're stuck providing the same interface to keep the app working, it does make sense to make these more simple when possible.

I'm somewhat optimistic that my REACH/AAA work will require some mobile reports and we'll be in a better position to simplify these more, but I don't htink that will be soon